manifests/install.pp

```diff
@@ -5,6 +5,10 @@
     package{ ['rubygem-smart_proxy_pulp']:
       ensure => installed,
     }
+
+    package{ ['katello-debug']:
+      ensure => installed,
```

Comment:
Actually why is this in the if block for pulp? We should always install `katello-debug`, or should we?